01-js-intro-and-jquery/keypoints/1_11/resources/script/script.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Created by MaximilianoDaniel on 11/09/2015.
+ */
+
+var onPageReady = function () {
+    var processRequest;
+    var showError;
+    var showComplete;
+    var listLenght;
+    var listAlbums;
+    var showAlbum;
+
+    showAlbum = function(album){
```

Comment:
spaces between function and parenthesis and also between the parenthesis and the curly brace